compile failure: dubbo-rpc-rest.jar: 2.7.1-SNAPSHOT not found

`
[ERROR] Failed to execute goal on project dubbo-samples-SPI-compatible: Could not resolve dependencies for project org.apache.dubbo:dubbo-samples-SPI-compatible:jar:1.0-SNAPSHOT: Failure to find org.apache.dubbo:dubbo-rpc-rest:jar:2.7.1-SNAPSHOT in https://repository.apache.org/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of apache.snapshots has elapsed or updates are forced -> [Help 1]
`

--- a/dubbo-samples-SPI-compatible/pom.xml
+++ b/dubbo-samples-SPI-compatible/pom.xml
@@ -30,7 +30,7 @@
         <target.level>1.8</target.level>
         <spring.version>4.3.16.RELEASE</spring.version>
         <dubbo.version>2.7.1</dubbo.version>
-        <dubbo.rpc.version>2.7.1-SNAPSHOT</dubbo.rpc.version>
+        <dubbo.rpc.version>2.7.1</dubbo.rpc.version>
         <zookeeper.version>3.4.13</zookeeper.version>
         <curator.version>4.0.1</curator.version>
         <validation-api.version>1.1.0.Final</validation-api.version>

